app-crypt/mcrypt/mcrypt-2.6.8.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Replacements for the old Unix crypt"
+DESCRIPTION="mcrypt is an enhanced replacement for the old crypt \
+command. mcrypt allows developers to use a wide range of encryption \
+functions without making drastic changes to their code. It contains the \
+actual encryption functions and provides a standardized mechanism for \
```

Comment:
This variant is also based on the description found in http://mcrypt.sourceforge.net/.
```diff
-DESCRIPTION="mcrypt is an enhanced replacement for the old crypt \
-command. mcrypt allows developers to use a wide range of encryption \
-functions without making drastic changes to their code. It contains the \
-actual encryption functions and provides a standardized mechanism for \
+DESCRIPTION="mcrypt is an enhanced replacement for the old crypt command. It \
+allows users to encrypt files or data streams without having to be \
+cryptographers. mcrypt uses libmcrypt, its companion library, which contains \
+the actual encryption functions and provides a standardized mechanism for \
 accessing them."